docs/_static/theme_overrides.css

```diff
@@ -11,3 +11,13 @@
       overflow: visible !important;
    }
 }
+
+/* Add padding to the parameters list on RTD theme */
+dl.simple dt {
+   padding-left: 4px;
+}
+
+dl.simple dt span.classifier {
+   padding-left: 6px;
+   font-weight: normal;
+}
```

Comment:
This CSS override is intended to add some spacing between parameters and make them more legible. In locally built docs the change is from:

<img width="594" alt="Screenshot 2019-07-11 09 02 38" src="https://user-images.githubusercontent.com/745966/61053049-ba847c00-a3ba-11e9-816e-32cb0ad07d4d.png">

To:

<img width="592" alt="Screenshot 2019-07-11 09 02 52" src="https://user-images.githubusercontent.com/745966/61053055-beb09980-a3ba-11e9-9be7-8800694f198e.png">

However, note that on our hosted website, this currently appears as follows:

<img width="675" alt="Screenshot 2019-07-11 09 04 27" src="https://user-images.githubusercontent.com/745966/61053111-e3a50c80-a3ba-11e9-8b84-1bfb064555f5.png">

So we will need to check and make sure this is correctly rendered by ReadTheDocs.org